RedHat-UX/red-hat-design-system
# docs: scroll padding permalinks

## What I did

1. Added a `scroll-padding-top` to the docs pages so that the permalinks don't appear behind the sticky navigation.


## Testing Instructions

1. Visit https://deploy-preview-1086--red-hat-design-system.netlify.app/elements/accordion/#overview
2. Verify that the `Overview` header is in view and not behind the header.

## Screenshots

Before, `Sample` header not visible:
![Screenshot 2023-06-23 at 2 20 10 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/3428964/b0c24ba8-de38-4701-a4cc-0326f12e9c8b)

After: `Sample` header is visible
![Screenshot 2023-06-23 at 2 20 32 PM](https://github.com/RedHat-UX/red-hat-design-system/assets/3428964/98a41e89-b6c5-4592-bc7b-e88f0c058e21)
